packages/amplication-data-service-generator/src/server/create-server.ts

```diff
@@ -65,8 +63,9 @@ export async function createServerModules(
   logger.info("Creating swagger...");
   const swaggerModule = await createSwagger();
 
+  const userEntity = entities.find((entity) => entity.name === "User");
```

Comment:
import const from packages/amplication-data-service-generator/src/server/user-entity.ts
```suggestion
  const userEntity = entities.find((entity) => entity.name === USER_ENTITY_NAME);
```